libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxDistanceFormatter.kt

```diff
@@ -136,25 +120,28 @@ class MapboxDistanceFormatter(
      * @param unit string from TurfConstants. This will be converted to the abbreviated form.
      * @return String with bolded distance and shrunken units
      */
-    private fun getDistanceString(distance: String, unit: String): SpannableString {
-        val spannableString = SpannableString("$distance ${unitStrings[unit]}")
+    fun getSpannableDistanceString(distanceAndSuffix: Pair<String, String>): SpannableString {
+        val spannableString = SpannableString("${distanceAndSuffix.first} ${distanceAndSuffix.second}")
 
         spannableString.setSpan(
-            StyleSpan(Typeface.BOLD),
-            0,
-            distance.length,
-            Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
+                StyleSpan(Typeface.BOLD),
```

Comment:
If you haven't already, setup `ktlint` when you have a chance to avoid these kind of changes. Setup instructions can be found here https://github.com/mapbox/mapbox-navigation-android/blob/master/docs/ktlint-setup-and-using.md and then `./gradlew ktlintFormat` and `⌥⌘L` are your friends 😅

Reply:
I set it up but I'll double check it.